### PR TITLE
Remove namespace field from ScheduleActivityTaskCommandAttributes message

### PR DESCRIPTION
**Pull request by alexshtin**
**What changed?**
Remove `namespace` field from `ScheduleActivityTaskCommandAttributes` message.

**Why?**
To remove ability of cross namespace activities calls. Initial motivation behind cross namespace activities was to allow different teams to own their own namespaces with activities and other teams to build workflows that use these activities in form of "library". Unfortunately, we found out that "cross namespace activity" is a wrong abstraction to achieve this goal. It adds complexity which is not well covered with test and not actually used (all official SDKs doesn't even expose it). For example, who should control activity heartbeat timeout?

And by the way, we are working on designing new feature/abstraction which will enable this functionality.

**How did you test it?**
N/A

**Potential risks**
Low risk: `namespace` field is not exposed for the most SDKs but there might be some unofficial SDK which will lose ability to start activity in another namespace.

### CODE REVIEW

**Comment by bergundy** [comment]
FTR, we removed `namespace` from TS `ActivityOptions` recently so AFAIK all SDKs no longer expose this

File: temporal/api/command/v1/message.proto [hunk 1]
```diff
@@ -44,7 +44,7 @@ import "temporal/api/taskqueue/v1/message.proto";
 message ScheduleActivityTaskCommandAttributes {
     string activity_id = 1;
     temporal.api.common.v1.ActivityType activity_type = 2;
-    string namespace = 3;
+    reserved 3;
```
Comment (bergundy) [inline temporal/api/command/v1/message.proto:47]: Let's add a comment explaining what was here before.

**Review by bergundy — APPROVED** [review]
Approved with a comment

**Comment by antstorm** [comment]
@alexshtin can you please expand on the "why?" of this change a bit more?

File: buf.yaml [hunk 2]
```diff
@@ -9,5 +9,7 @@ lint:
     - DEFAULT
 breaking:
   ignore:
+    - temporal/api/command/v1
+    - temporal/api/history/v1
```
Comment (cretz) [inline buf.yaml:13]: I assume you'll come back in a separate PR and remove these two lines

**Review by cretz — APPROVED** [review]

File: buf.yaml (hunk 2 above)
Comment (alexshtin) [reply buf.yaml:13]: yes!

**Comment by bergundy** [comment]
> @alexshtin can you please expand on the "why?" of this change a bit more?

We want to support cross namespace calls via another mechanism, one that doesn't require the caller to set options which are implementation details of the activity (e.g. timeouts and retry policy).

**Comment by alexshtin** [comment]
> @alexshtin can you please expand on the "why?" of this change a bit more?

Updated. Does PHP SDK expose cross namespace activities?

**Pull request merged**